index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>José Fernández-Caminero: Full-Stack Engineer; Portfolio</title>
+    <link rel="stylesheet" href="css/styles.css">
+</head>
+<body>
+    <nav>
+        <img class="artdeco1" src="resources/image_geometry_1.svg" alt="Geometrical decoration">
+        <img src="resources/ic_menu.svg" alt="Toolbar menu">
+    </nav>
+
+    <h1>José Fernández-Caminero</h1>
+
+    <p>I love Full-Stack Engineering and I can code websites that are both the most complex and the most efficient.</p>
+
+    <button>Start collaboration</button>
+
+    <footer>
+        <img src="resources/image_geometry_2.svg" alt="Geometrical decoration 2">
+    </footer>
```

Comment:
- [ ] This is the headline section so it would be better if you wrap it with the `<section>` tag in order to make your structure makes sense and clean